README.md

```diff
@@ -1,9 +1,17 @@
 # The Fundamental Theorem of Everything!
 
+
+## Arithmetic
+Every integer greater than 1 can be uniquely factored into prime numbers up to the order of factors.
+
 ## Algebra
 
 Every non-zero, single-variable polynomial of degree n with complex coefficients has exactly n complex roots counted with multiplicity.
 
+## Calculus
+
+Let F be an antiderivative of f. Then (integration of f from a to b) = F(b) - F(a)
```

Comment:
https://en.wikipedia.org/wiki/Fundamental_theorem_of_calculus#Formal_statements

Please, consider the first and the second fundamental theorems of calculus and make more formal.

Thank you!